velum/impl/gateway.py

```diff
@@ -180,11 +182,12 @@ def _raise_for_unhandled_message(self, message: "_WSMessage") -> typing.NoReturn
 
 
 class GatewayHandler(gateway_trait.GatewayHandler):
-
     __slots__ = (
```

Comment:
Just out of curiosity-- what's with the sorting here? These used to be alphabetical (or so I'd like to believe). Mostly just asking as it seems like `_connection_event` seems to have been intentionally moved.